src/controllers/task.list.controller.ts

```diff
@@ -0,0 +1,15 @@
+import { Request, Response } from "express";
+import { Templates } from "../types/template.paths";
+import { initTaskList, TaskList } from "../types/task.list";
+import { CompanyProfile } from "@companieshouse/api-sdk-node/dist/services/company-profile/types";
+import { getCompanyProfile } from "../services/company.profile.service";
+
+export const get = async (req: Request, res: Response) => {
+  const companyNumber = req.params.companyNumber as string;
```

Comment:
there is a urlParams.PARAM_COMPANY_NUMBER that can be used to identify the params name - req.params[urlParams.PARAM_COMPANY_NUMBER]

Reply:
I'm sure that amounts to the same thing but will change this if it is preferred.

Reply:
it will be the same value but the benefit of using constants is that we only need to to change it in one place if it needs to change as opposed to searching out all occurrences of req.params.companyNumber.